src/library_gl.js

```diff
@@ -475,14 +475,18 @@ var LibraryGL = {
           sizePerPixel = numChannels*1;
           break;
         case 0x1403 /* GL_UNSIGNED_SHORT */:
+#if USE_WEBGL2
+        case 0x140B /* GL_HALF_FLOAT */:
+#else
         case 0x8D61 /* GL_HALF_FLOAT_OES */:
```

Comment:
Isn't it promoted in this spec? I thought `GL_HALF_FLOAT_OES` was supposed to error out in GLES3 spec. But I can't find where I saw that...